packages/veritone-react-common/src/components/ObjectDetectionEngineOutput/index.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component } from 'react';
+import { arrayOf, shape, number, string, func } from 'prop-types';
+import classNames from 'classnames';
+import { kebabCase } from 'lodash';
+
+import EngineOutputHeader from '../EngineOutputHeader';
+import PillButton from '../share-components/buttons/PillButton';
+import { msToReadableString } from '../../helpers/time';
+
+import styles from './styles.scss';
+
+const ObjectGroup = ({
```

Comment:
let's try to stick to one component per file as a rule of thumb, break this out into it's own file.